webui/tests/tests/StepImplementation.ts

```diff
@@ -11,6 +11,7 @@ import {
 /* eslint-enable no-unused-vars */
 
 import * as assert from 'assert';
+import * as expect from 'expect';
```

Comment:
this is great to have, as taiko does not natively have it (which is fine because there are so many libraries for this)